spec/models/procurement_area_spec.rb

```diff
@@ -4,6 +4,10 @@
   it { should validate_presence_of(:name) }
 end
 
+RSpec.describe ProcurementArea, "relationships" do
+  it { should have_many(:rota_slots).dependent(:destroy) }
```

Comment:
Can we change this to be an explicit test that creates a ProcurementArea with RotaSlots, then deletes the ProcurementArea and asserts that the RotaSlots are also removed.
That way the test is implementation independent and not reliant on the shoulda matchers